### PR TITLE
Add uzb translations to 4.4

### PR DESCRIPTION
**Pull request by shokhaa**
| Q             | A
| ------------- | ---
| Branch?       | 5.4 for features / 4.4, 5.2 or 5.3 for bug fixes 
| Bug fix?      | no
| New feature?  | yes 
| Deprecations? | no 
| Tickets       | Fix #... 
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+                <target>Qiymat quyidagi cheklovlardan kamida bittasiga javob berishi kerak:</target>
+            </trans-unit>
+            <trans-unit id="98">
+                <source>Each element of this collection should satisfy its own set of constraints.</source>
+                <target>Ushbu to'plamdagi har bir narsa o'ziga xos cheklovlarni qondirishi kerak.</target>
+            </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid International Securities Identification Number (ISIN).</source>
+                <target>Qiymat Qimmatli qog'ozlarning xalqaro identifikatsiya raqami (ISIN) ga mos emas.</target>
+            </trans-unit>
```
Comment (fabpot) [inline src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf:388]: There is now an additional one (100). Can you add it as well?

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf (hunk 1 above)
Comment (shokhaa) [reply src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf:388]: @fabpot certainly can! but I did not see from the English translations that there are 100

Comment (fabpot) [reply src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf:388]: Most translation already exists. You should rebase your 4.4 branch to the current 4.4 branch from symfony/symfony. That would also get you the new translation.

Comment (shokhaa) [reply src/Symfony/Component/Validator/Resources/translations/validators.uz.xlf:388]: @fabpot yes I saw! then you can delete this pull request, and I'll fix it and create a new one!

**Pull request closed without merging**